### DISCUSSION TITLE
Add file upload component to design system

### DISCUSSION THREAD

**Discussion by joshmfranklin** [CMSgov/design-system#1821] [Ideas]
## What
Our project MAC-MCRRS is starting the process of designing/building a multi-step intake form. As part of that process users will need to upload documents. We've been using the USWDS file upload component in our design thus far.

## Why
- A file upload is a common process in many transactional services between two parties
- A file upload component will help standardize how an upload is triggered and how the chosen file will display
- [USWDS](https://designsystem.digital.gov/components/form-controls/#file-input) has set some precedence for file upload

**Comment by line47** [comment]
https://jira.cms.gov/browse/WNMGDS-923
